examples/dbclient/pokemons/src/main/resources/application.yaml

```diff
@@ -46,7 +46,7 @@ db:
     #
 #    url: "jdbc:h2:tcp://localhost:9092/~test"
 #    username: h2
-#    password: "${EMPTY}"
+#    password: ""
```

Comment:
Why would you change it like this? We mus use ${EMPTY} for empty strings

Reply:
Removed as irrelevant file.